docs/keys.md

```diff
@@ -416,6 +416,9 @@ m.route(rootElem, "/", {
 })
 ```
 
+Note we've added the `key` property to the child of `Layout`,
+i.e. it's `m(Layout, m(Person, {key: ...}))` not `m(Layout, {key: ...}, m(Person, ...))`.
```

Comment:
What exactly is the intent here? I'm not sure how this applies in or out of context. (Also, Latin abbreviations like "i.e." isn't as widely known among non-native speakers, so it's best to avoid those in general in the docs.)